db/data_migration/20160208121238_republish_statistics_announcements_to_publishing_api.rb

```diff
@@ -0,0 +1,4 @@
+StatisticsAnnouncement.where.not(content_id: "").find_each(batch_size: 100) do |statistics_announcement|
+  statistics_announcement.publish_to_publishing_api
```

Comment:
I've [commented on the implementation](https://github.com/alphagov/whitehall/pull/2457/files#r52496738). If we keep the implementation of that, then this will also push every Statistics Announcement to the search index, which is unnecessary. If we used the `PublishingApiRepublisher` instead, that isn't a concern:
```ruby
statistics_announcements = StatisticsAnnouncement.where.not(content_id: "")
DataHygiene::PublishingApiRepublisher.new(statistics_announcements)
```